2-mandatory/1-create-functions.js

```diff
@@ -44,8 +52,24 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(array) {
+let input = [];
```

Comment:
You don't have to define an empty array. Mapping on array instead of traditional for loop is a neater option.